### ISSUE TITLE
Upgrade to Laravel 5.7 to use Socialite with People api instead of Google+ 

### ISSUE THREAD

**Issue by arborrow** [arborrow/montserrat#269]
Currently our login check is dependent upon Google+ API which is being deprecated in March 2019 (I believe). I would like to confirm that we can disable the Google+ API and still use Socialite to login. I read one article at: https://laravel-news.com/google-api-socialite that did not mention Google+ but it would be helpful to have a better understanding of how we will be authenticating moving forward.

**Comment by arborrow** [comment]
I have marked this as a critical bug because of its importance even though currently there is no bug per se

**Comment by arborrow** [comment]
OK, from the composer.json file it appears that Polanco was using socialite version 3. I just noticed a commit in version 3 from about 11 days ago that I suspect will address this issue - https://github.com/laravel/socialite/commit/4d24de35e531513bb23a08dcc5e9ddeea867b892 

We will want to test with a development site first and then if that works update composer and then disable the Google+ api to verify that login works properly.

**Comment by arborrow** [comment]
FYI, I have enabled the people API so that it is ready to go when the code just changed. I have commented on the pull request that a new tag needs to be created that includes the latest changes. I trust these will be made before the Google+ API goes bye-bye.

**Comment by arborrow** [comment]
The other option is to upgrade Laravel to 5.7 which uses socialite 4.0. See https://laravel.com/docs/5.7/upgrade for details. I think this may be the best route to get us updated with 5.7 and use the people api instead of Google+.

**Comment by arborrow** [comment]
If we update composer.json to use socialite version 3.0.x-dev then it will implement people api instead of Google+. I have implemented this as part of #263 and it seems to work well. We may need to update the documentation to reflect the change away from Google+ api.  One change I noticed was the size of the img file was much bigger so I specified the template to display the image at 75x75 (slightly larger that the 50x50 image before).

**Comment by arborrow** [comment]
Just to document, this has to do with checking the socialite domain limitation which was not enable on production. The error I was seeing on production is: Client error: `POST https://accounts.google.com/o/oauth2/token` resulted in a `400 Bad Request` response: { "error": "invalid_grant", "error_description": "Bad Request" }. So will need to see what is causing things to get tripped up. I read https://stackoverflow.com/questions/48616799/laravel-socialite-400-bad-request-response but was too tired to really process what it was saying about the access code.

**Comment by arborrow** [comment]
I have removed this as being a critical bug and made it a simple improvement. I have made the changes to use Google People API and disabled the Google+ API. Keep in mind that other branches (development - including master) that were using socialite's 3.x version will no longer work with montserratretreat.org as the Google+ API has been disabled. Also, keep in mind that the user array in the People API refers to the domain as 'hd' rather than 'domain' (as was returned by the G+ API).
